bin/i18n/resources/dashboard/course_content/sync_out.rb

```diff
@@ -33,7 +33,8 @@ def restore_level_content(file_subpath, crowdin_file_path)
             return false unless File.exist?(original_file_path)
 
             # Course content should be merged with existing content, so existing data doesn't get lost
-            restored_i18n_data = RedactRestoreUtils.restore_file(original_file_path, crowdin_file_path, REDACT_PLUGINS)
+            restored_i18n_file = RedactRestoreUtils.restore_file(original_file_path, crowdin_file_path, REDACT_PLUGINS)
+            restored_i18n_data = JSON.parse(restored_i18n_file)
```

Comment:
Because in the changes I introduced, `redact_file` and `restore_file` methods don't assume the file content is in JSON format, we need to parse the restore content.